app/controllers/hosts_controller.rb

```diff
@@ -912,7 +912,7 @@ def host_attributes_for_templates(host)
   end
 
   def csv_columns
-    [:name, :operatingsystem, :compute_resource_or_model, :hostgroup, :last_report]
+    User.current.table_preferences.find_by(name: controller_name)&.columns&.map { |col| col == 'os_title' ? 'operatingsystem' : col } - ['power_status']
```

Comment:
Why is power status hardcoded to be excluded?

Reply:
I don't think that it's something necessary to be exported. And it always stays in the first column in exported file which tbh does not look good.

Reply:
Make sense, power column is now included in exported file.